src/pages/profile/create/index.tsx

```diff
@@ -9,51 +9,95 @@ import TextField from '@mui/material/TextField';
+  DEFAULT_FEMALE_IMAGE,
+  DEFAULT_LANGUAGE,
+  DEFAULT_MALE_IMAGE,
+  PHONE_CODES_BY_COUNTRY,
+} from '@src/constants';
+import { PhoneCodesByCountry } from '@src/constants/constant.types';
+import { CreateProfileForm, ErrorTypes, ProfileActions } from '@src/types/i18n.types';
 import { Gender, Profile, ThemeMode } from '@src/types/profile.types';
 import { useTranslation } from 'react-i18next';
 import AddImage from '../AddImage';
-import { DEFAULT_COUNTRY, PHONE_CODES_BY_COUNTRY } from '../constants';
-import { PhoneCodesByCountry } from '../constants.types';
 import UserFullName from './UserFullName';
 import UserPhoneNumber from './UserPhoneNumber';
 
 function CreateProfile() {
-  const { t } = useTranslation();
   const {
     register,
     handleSubmit,
     formState: { errors },
     reset,
     control,
   } = useForm<Profile>();
-  const defaultMaleImage =
-    'https://i.pinimg.com/564x/48/6c/a0/486ca00640b169300b48e9ceacd8e401.jpg';
-  const defaultFemaleImage =
-    'https://i.pinimg.com/564x/39/42/01/39420149269ede36847932935b26f0b8.jpg';
 
-  const [imageUrl, setImageUrl] = useState('');
   const [selectedCountry, setSelectedCountry] = useState<PhoneCodesByCountry>(DEFAULT_COUNTRY);
-  const { mutate, isPending } = useCreateAccountMutation();
+  const [profileImage, setProfileImage] = useState<File | null>(null);
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  const { mutate: createAccountMutation, isPending: profileCreatePending } =
+    useCreateAccountMutation();
+  const { mutate: updateAccountImage, isPending: imageUpdatePending } =
+    useUpdateAccountImageMutation(profileImage);
+
+  const confirmCreation = async () => {
+    navigate(ROUTES.root);
+    toast.success(t(ProfileActions.profile_create));
+    await queryClient.invalidateQueries({ queryKey: [QUERY_KEYS.query.user] });
+  };
 
   const onSubmit: SubmitHandler<Profile> = (data) => {
-    const defaultImage = data.gender === Gender.male ? defaultMaleImage : defaultFemaleImage;
+    const defaultImage = data.gender === Gender.male ? DEFAULT_MALE_IMAGE : DEFAULT_FEMALE_IMAGE;
 
-    const userProfile: Profile = {
+    const profileData: Profile = {
       ...data,
       phoneNumber: selectedCountry.code + data.phoneNumber,
-      imageUrl: defaultImage,
-      language: DEFAULT_LANGUAGE,
+      language: DEFAULT_LANGUAGE.code,
       uiTheme: ThemeMode.light,
     };
 
-    mutate(userProfile);
+    try {
+      if (profileImage) {
+        createAccountMutation(profileData, {
+          onSuccess: (res) => {
+            updateAccountImage(res.id, {
+              onSuccess: () => {
+                confirmCreation().catch((error) => console.error(error));
+              },
+            });
+          },
+        });
+      } else {
+        createAccountMutation(
+          { ...profileData, imageUrl: defaultImage },
+          {
+            onSuccess: () => {
+              confirmCreation().catch((error) => console.error(error));
```

Comment:
remove console.errors, make real notification instead, moreover you can use short notation
instead
confirmCreation().catch((error) => console.error(error));
write in such manner
confirmCreation().catch(console.error);